video_creation/background.py

```diff
@@ -64,12 +64,12 @@ def chop_background_video(video_length, vidpath):
         noerror = False
         return noerror
     start_time, end_time = get_start_and_end_times(video_length, background.duration)
-    ffmpeg_extract_subclip(
-        f"assets/mp4/{vidpath}.mp4",
-        start_time,
-        end_time,
-        targetname="assets/mp4/clip.mp4",
-    )
+    
+    with VideoFileClip("assets/mp4/background.mp4") as video:
```

Comment:
Please make the path an f-string with {vidpath} instead of background - other than that it looks fine.